gradle/testing.gradle

```diff
@@ -217,6 +217,12 @@ tasks.withType(Test).configureEach { task ->
             systemProperties['tests.iterations'] = project.getProperty('tests.iterations')
         }
         task.testFramework.options.excludeTags.add('Performance')
+
+        if (task.name == 'test') {
```

Comment:
I don't think we want to limit this, I think we want all test tasks (at least for now), to set these properties.

Reply:
Hm, maybe we should revisit this when we set `paralle.enabled` to `true`. In the future, I think we want the `test` task to be executed in parallel, but not the `destructiveTest` or `performanceTest`. In theory, we could actually do `destructiveTest` with a `ResourcLock` to ensure that two tests that both wipe FDB don't execute in parallel.

Reply:
I suspect that a `ResourceLock` would effectively remove any parallelism anyways. And I think a `ResourceLock` only applies if they are in the same JVM.
Any test in `destructiveTest` would be expected to wipe the database, and thus they all would have the same resource lock, and if not, they can be in the regular `test`.